lib/SILGen/SILGenLValue.cpp

```diff
@@ -2639,6 +2647,8 @@ namespace {
         AccessKind(accessKind) {}
 
     void emitUsingStrategy(AccessStrategy strategy) {
+      auto var = dyn_cast<VarDecl>(Storage);
```

Comment:
doesn't seem used?

Reply:
Ah, I might have not cherry-picked NFC commit, thank you!

Reply:
Removed!